homeassistant/components/velbus/config_flow.py

```diff
@@ -75,13 +77,3 @@ async def async_step_user(self, user_input=None):
             ),
             errors=self._errors,
         )
-
-    async def async_step_import(self, user_input=None):
```

Comment:
We still have the import from `async_setup` in the init module. Our standard is to keep the import step until we invalidate the config YAML.